include/api.tickets.php

```diff
@@ -161,6 +161,91 @@ function processEmail($data=false) {
         return $this->createTicket($data);
     }
 
+    function search($format) {
+      $this->initalizeUser($format);
+      $params = $this->getRequest($format);
+
```

Comment:
did not read all the code, but anyway here is something wrong, what must be $search?